CHANGES_NEXT_RELEASE

```diff
@@ -36,6 +36,7 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Add: q-string filters for compound values of attributes (Issue #1156)
```

Comment:
I understand you refer to "free search without knowing the path". Finally, that is not part of #1156 scope, see: https://github.com/telefonicaid/fiware-orion/issues/1156#issuecomment-233252369

> (*) For the parts not covered in the current NGSIv2 spec (e.g. free search without knowing the path) a new issue should be created.